worlds/sc2/rules.py

```diff
@@ -1030,7 +1030,7 @@ def zerg_infested_tank_with_ammo(self, state: CollectionState) -> bool:
                 state.has(item_names.INFESTED_SIEGE_TANK, self.player)
                 and (
                         state.has_all({item_names.INFESTOR, item_names.INFESTOR_INFESTED_TERRAN}, self.player)
-                        or state.has(item_names.INFESTED_BUNKER, self.player)
+                        or state.has_any((item_names.INFESTED_BUNKER, item_names.INFESTED_MARINE), self.player)
```

Comment:
Maybe advanced tactics as you need to spend minerals per shot and it's more difficult to rally those to tanks at different spots